Tests/pfio_MAPL_demo.F90

```diff
@@ -74,9 +74,10 @@ program main
 !BOC
 
       ! Read and parse the command line, and set parameters
-      cli = MAPL_FlapCLI(description = 'GEOS AGCM', &
-                         authors     = 'GMAO')
-      cap_options = MAPL_CapOptions(cli)
+      cap_options = MAPL_FlapCLI( &
```

Comment:
Ooooooh. This might mean this is only for MAPL3. Otherwise we have to change things like `GEOSgcm.F90` (see https://github.com/GEOS-ESM/GEOSgcm_App/blob/develop/GEOSgcm.F90)

Is there a way to do this and not to change the interface?

Reply:
this is backward compatible. Notice the difference between GEOSgcm:
```   
cli = MAPL_FlapCLI(description = 'GEOS AGCM', &
                                     authors     = 'GMAO')
   cap_options = MAPL_CapOptions(cli)
```

and pfio that only uses cap_options = MAPL_CapOptions(cli)

Reply:
I asked @Gvilla1000-nasa about this privately.   He claims this version works with GEOSgcm.F90.

Reply:
Ohh. Cool. Well, if it does, then I guess no blocking label needed. I'm all for changes in MAPL that don't need anything outside MAPL.

@bena-nasa Query: Do all bits of GEOS that use FLAP (including any in the ADAS if there are any?) use this sort of initialization of FLAP?

Reply:
to elaborate a bit more. MAPL_FlapCLI is an interface to two different functions. One whose output is of MAPL_FlapCLI type and another whose output is of MAPL_CapOptions type. The new version is the latter but to prevent errors during compilation time because of the module procedures having the same number of argument, I was forced to add a dummy argument to the new version.

Reply:
It's all a bit jury rigged at the moment to preserve backward compatibility.   This is probably a good time to create a MAPL 3 branch for GEOSgcm_App and change MAPL to only have the desired interfaces.